packages/charts/src/charts/table/header-row.component.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import { HeaderCeil, ArrowsType, SortByType } from '@keen.io/ui-core';
+import { FormatFuncType } from '../../types';
+import { Header } from './table.styles';
+
+type Props = {
+  data?: any;
+  color: string;
+  onClick?: (res: { property: string; sort: ArrowsType }) => void;
+  sorting?: SortByType;
+  format?: FormatFuncType;
+  onResize?: (res: { property: string; width: number }) => void;
+};
+
+export const HeaderRow = ({
+  data,
+  color,
+  onClick,
+  sorting,
+  format,
+  onResize,
+}: Props) => {
+  const [dragged, setDragged] = useState(false);
+
+  return (
+    <Header color={color}>
+      {Object.keys(data).map(key => (
+        <HeaderCeil
+          key={`${key}`}
+          onClick={onClick}
+          sorting={sorting}
+          format={typeof format === 'object' ? format[key] : format}
```

Comment:
In `keen.io/dataviz` there is already `mappings` feature - that allow to map / transform labels on higher level. Please, check it - because from my perspective this logic is duplicated here.
I'll leave format as function to format specific columns - based on their key property.